R/dotdotdot.R

```diff
@@ -79,3 +79,13 @@ unpack_bool_expr_result = function(...) {
       }
     })
 }
+
+
+#' Convert dots to a character vector of column names
+#' @param .df [RPolarsDataFrame]
+#' @param ... Arguments to pass to [`pl$col()`][pl_col]
+#' @noRd
+dots_to_colnames = function(.df, ..., .call = sys.call(1L)) {
+  result(pl$DataFrame(schema = .df$schema)$select(pl$col(...))$columns) |>
+    unwrap(call = .call)
+}
```

Comment:
I'm surprised this wasn't needed before , I think `DataFrame$drop()` should have a similar input as `DataFrame$partition_by()` based on the py-polars API

Reply:
@etiennebacher Could you update `$drop()`?